### ISSUE TITLE
Color contrast does not meet minimum requirement (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#141]
313,002 occurrences but this should be one or two small changes in the CSS. One problem is:

- green font used in the diplomatic and normalized versions to show legibility, deletions, etc. 
- the footer links are red and do not provide sufficient contrast with the taupe background color
- there are more ... this will have to be fixed and rescanned in SiteImprove to reduce the occurrences to more easily pin point where the fixes need to be made

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [x] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow.

**Comment by tubesoft** [comment]
I am changing the text color of #018B84 to #007F79, which improves the contrast from 3.86:1 to 4.503711010093676.
The color is just becoming a little bit darker.

**Comment by mdalmau** [comment]
@tubesoft : are you verifying the changes with the SiteImprove browser plugin? I am just curious if you are verifying local fixes with the plugin. Thanks!

**Comment by tubesoft** [comment]
Another issue is the background of the footer.
I am changing it from #BCAB81 to #DFD9BD, which is the same color as the header's background.
This improves the contrast from 3.94 to 6.284904266298057.

<img width="599" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/d0e406d5-5bc6-47e0-8fb5-8cb69a25c232">

<img width="996" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/3a534999-89e1-470d-9a64-88bcca9beea4">

After:
<img width="1197" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/65a64b9d-1991-4832-9870-9de0989da099">

**Comment by tubesoft** [comment]
@mdalmau I actually googled a color contrast checker on the web!

**Comment by tubesoft** [comment]
@mdalmau SiteImprove also lists "Enhanced" color contrast requirement. Do I have to work on that too? Or is it optional?

**Comment by mdalmau** [comment]
@tubesoft I am not sure what you mean. Can you include a screen shot for "enhanced" color contrast? Thanks!

**Comment by tubesoft** [comment]
@mdalmau This is it:
<img width="1082" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/7775a45d-2f6d-4d1c-bebb-96b6ab6dd2db">

**Comment by mdalmau** [comment]
@tubesoft : No -- We only need to fix A and AA issues.

**Comment by tubesoft** [comment]
@mdalmau Good to know that! Thanks.

**Comment by tubesoft** [comment]
@mdalmau, I fixed the issues and committed, as you can see above.

**Comment by mdalmau** [comment]
Since this issue is now part of https://github.com/IUBLibTech/newton_chymistry/pull/146, I will mark this issue as "done" so I can better see the remaining accessibility issues!
